### DISCUSSION TITLE
Page 404 is not called after generating static files with next export

### DISCUSSION THREAD

**Discussion by henriquematiasb** [vercel/next.js#23206] [Help]
Hi everybody!

When using static file generation and opening the project index, I noticed that page 404 is not triggered when accessing a non-existent route.

### Here is the step by step to reproduce the problem:

- Run "next export"

- Access the "/ out" folder

- Open the "index.html" file

- Try to access a page that does not exist. You’ll see that 404 isn’t called

#### **Any solution ?**

**Comment by akellbl4** [comment]
Hi!
When you make `next export` your site became absolutely static and you should serve it with some kind of server like `nginx`
By default, all servers will show you their own 404 page. You should configure it for showing your 404 page.
